spec/features/admin/orders_spec.rb

```diff
@@ -267,7 +267,7 @@ def new_order_with_distribution(distributor, order_cycle)
       end
 
       scenario "shows the order tax adjustments" do
-        within('tbody#order-charges') do
+        within('fieldset', text: 'LINE ITEM ADJUSTMENTS') do
```

Comment:
We should reference the translation key instead as per https://github.com/openfoodfoundation/openfoodnetwork/wiki/Code-Conventions#always-use-i18nt-in-spec-assertions, right?

Reply:
I thought about that. But then this was much easier for several reasons:

- The translation key was missing. While rendering of the page has a fallback and uses the key name, that doesn't happen when calling `I18n.t` in the spec. I added the missing translation now and could use it.
- Some Spree logic (maybe the view) is printing it all in capitals. So I would need to use this: `I18n.t('spree.admin.orders.form.line_item_adjustments').upcase` which is long and hard to read.

But we have a convention, so I better obey. :wink:

Reply:
🤣